modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_set.scss

```diff
@@ -50,6 +50,26 @@ body {
 
 // Buttons
 
+.btn-danger {
+	@include button-cp(
```

Comment:
The whole `-cp` and `-cpp` is starting to annoy me... not for this PR, and we should've probably caught it sooner.

I'd say we better avoid abbreviations like this in the future, since it makes the code so much harder to decipher.

Reply:
After all... naming is the hardest thing in Computer Science 😉

Reply:
I think this come from the first version of the layer, in code environment I prefer ccp as prefix and suffix instead of css-custom-properties

Reply:
I can never, and I mean never, remember correctly what ccp means. I've had at least 3 or 4 people asking what this means at all, so I don't think neither `-cp` nor `-ccp` are helping us make sense of the code.

Options:

```
@include custom-properties-button(...);
@include button-custom-properties(...);
...
```

No-one has to wonder what is `-cp` ever again :)